core/src/main/resources/config.yml

```diff
@@ -36,6 +36,15 @@ anti-bot:
 # For more information: https://github.com/games647/FastLogin#why-do-players-have-to-invoke-a-command
 autoRegister: false
 
+# Should FastLogin respect per IP limit of registrations (e.g. in AuthMe)
+# Because most auth plugins do their stuff async - FastLogin will still think the player was registered
+# To work best - you also need to enable auto-register-unknown
```

Comment:
`auto-register-unknown` You meant `auto-register`?

Reply:
Nope, I meant `auto-register-unknown` - that way FastLogin will try to register the player again, so the player would get the message about the IP limit again. If `auto-register-unknown` is set to false - they would only get that message the first time they join and they might not see it immediately

If `auto-register` is set to false - the IP limit would just not be used by FastLogin, because it wouldn't register the player at all